charts/target-group-binding/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: aws-target-group-binding
+description: A Helm chart for deploying a TargetGroupBinding for use with the AWS LoadBalancer Controller
+type: application
+version: 0.1.2
```

Comment:
This is probably still version 0.1.0 as this is the first release?

Reply:
put back down